src/analyze.ts

```diff
@@ -93,6 +95,10 @@ const staticModules = Object.assign(Object.create(null), {
     default: os,
     ...os
   },
+  '@mapbox/node-pre-gyp': {
+    default: mapboxPregyp,
+    ...mapboxPregyp
+  },
   'node-pre-gyp': pregyp,
   'node-pre-gyp/lib/pre-binding': pregyp,
   'node-pre-gyp/lib/pre-binding.js': pregyp,
```

Comment:
Should we replace the legacy `node-pre-gyp` with the new mapbox one here or keep them both?

Reply:
We can keep both for legacy compat, possibly dropping the old one in a subsequent major.